tests/cargo-kani/ws-specified/bincrate/src/main.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! This test's purpose is to ensure that we successfully build
```

Comment:
This comment is from the other test, and does not reflect what the test is checking. Can you please update it?

Reply:
Fixed with 64225f0ecb2ffce2c4176c307d9905abb0f16fc7